modules/private-clusters-setting-api-private.adoc

```diff
@@ -37,48 +38,66 @@ ifndef::cpmso-using-azure[]
 *** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
 endif::cpmso-using-azure[]
 ifndef::cpmso-using-aws[]
-*** For Azure, delete the `api-internal` rule for the load balancer.
+*** For Azure, delete the `api-internal-v4` rule for the public load balancer.
+endif::cpmso-using-aws[]
+
+ifndef::cpmso-using-aws[]
+.. For the Azure private load balancer, if you configure the Ingress Controller endpoint publishing scope to `Internal` you must create a rule called `outbound-rule-v4` that connects the public load balancer to the backend address pool.
```

Comment:
The action is done in public load balancer, so this is for Azure public load balancer, not private one.

I think the rule name `outbound-rule-v4` is not important here, but need to indicate that the type of created rule is outbound rule to provide outbound traffic for backend instances.